nornir_netbox/plugins/inventory/netbox.py

```diff
@@ -308,22 +310,32 @@ def load(self) -> Inventory:
         groups = Groups()
         defaults = Defaults()
 
-        if self.defaults_file.exists():
-            with self.defaults_file.open("r") as f:
-                defaults_dict = yml.load(f) or {}
-            defaults = _get_defaults(defaults_dict)
-        else:
-            defaults = Defaults()
+        try:
+            if self.defaults_file.exists():
+                with self.defaults_file.open("r") as f:
+                    defaults_dict = yml.load(f) or {}
+                defaults = _get_defaults(defaults_dict)
+        except PermissionError as e:
+            if not self.ignore_file_errors:
+                raise PermissionError(e)
 
-        if self.group_file.exists():
-            with self.group_file.open("r") as f:
-                groups_dict = yml.load(f) or {}
+            warnings.warn("Error ignored while reading file: {}".format(e))
```

Comment:
Can we use the logging library here instead?
The message can be more explicit, something like `Unable to read {file} due to a permission issue`